### ISSUE TITLE
Error compiling on Slackware 15.0

### ISSUE THREAD

**Issue by sateuwdie** [HappySeaFox/sail#220]
Compiling on Slackware 15.0 give me this error

```
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__atof':
svg.c:(.text+0x63b): undefined reference to `pow'
/usr/bin/ld: svg.c:(.text+0x75b): undefined reference to `pow'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__parseColor':
svg.c:(.text+0xbdb): undefined reference to `roundf'
/usr/bin/ld: svg.c:(.text+0xbf2): undefined reference to `roundf'
/usr/bin/ld: svg.c:(.text+0xc25): undefined reference to `roundf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__addShape':
svg.c:(.text+0x1349): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0x1360): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__curveBounds':
svg.c:(.text+0x1a9c): undefined reference to `sqrt'
/usr/bin/ld: svg.c:(.text+0x1ba8): undefined reference to `sqrt'
/usr/bin/ld: svg.c:(.text+0x1c88): undefined reference to `sqrt'
/usr/bin/ld: svg.c:(.text+0x1d50): undefined reference to `sqrt'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__rasterizeSortedEdges':
svg.c:(.text+0x30e5): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__prepareStroke.isra.0':
svg.c:(.text+0x4d00): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__parseTransformArgs.constprop.0':
svg.c:(.text+0x5e75): undefined reference to `pow'
/usr/bin/ld: svg.c:(.text+0x5ff0): undefined reference to `pow'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__createGradient':
svg.c:(.text+0x6551): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__parseCoordinateRaw':
svg.c:(.text+0x6660): undefined reference to `pow'
/usr/bin/ld: svg.c:(.text+0x6827): undefined reference to `pow'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__parseTransform':
svg.c:(.text+0x6ad2): undefined reference to `tanf'
/usr/bin/ld: svg.c:(.text+0x6d40): undefined reference to `sincosf'
/usr/bin/ld: svg.c:(.text+0x6ecc): undefined reference to `tanf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__parseAttr':
svg.c:(.text+0x7964): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0x7adb): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0x7af0): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0x7b0a): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__roundCap.isra.0':
svg.c:(.text+0x86b1): undefined reference to `sincosf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__expandStroke':
svg.c:(.text+0x8ab5): undefined reference to `acosf'
/usr/bin/ld: svg.c:(.text+0x98e9): undefined reference to `atan2f'
/usr/bin/ld: svg.c:(.text+0x990b): undefined reference to `atan2f'
/usr/bin/ld: svg.c:(.text+0x9bc6): undefined reference to `sincosf'
/usr/bin/ld: svg.c:(.text+0xaeae): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xaef6): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xaf42): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvg__startElement':
svg.c:(.text+0xc4c9): undefined reference to `sincosf'
/usr/bin/ld: svg.c:(.text+0xc78e): undefined reference to `acosf'
/usr/bin/ld: svg.c:(.text+0xc859): undefined reference to `acosf'
/usr/bin/ld: svg.c:(.text+0xc9a2): undefined reference to `sincosf'
/usr/bin/ld: svg.c:(.text+0xd82c): undefined reference to `sincosf'
/usr/bin/ld: svg.c:(.text+0xd95e): undefined reference to `acosf'
/usr/bin/ld: svg.c:(.text+0xdbda): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xde7b): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xdebb): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xdef2): undefined reference to `sqrtf'
/usr/bin/ld: svg.c:(.text+0xdf0e): undefined reference to `sqrtf'
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o:svg.c:(.text+0xdf21): more undefined references to `sqrtf' follow
/usr/bin/ld: CMakeFiles/sail-codec-svg.dir/svg.c.o: in function `nsvgRasterize':
svg.c:(.text+0x1028f): undefined reference to `fmodf'
/usr/bin/ld: svg.c:(.text+0x10868): undefined reference to `sqrtf'
collect2: error: ld returned 1 exit status
make[2]: *** [src/sail-codecs/svg/CMakeFiles/sail-codec-svg.dir/build.make:98: src/sail-codecs/svg/sail-codec-svg.so] Errore 1
make[1]: *** [CMakeFiles/Makefile2:1802: src/sail-codecs/svg/CMakeFiles/sail-codec-svg.dir/all] Errore 2
make: *** [Makefile:146: all] Errore 2
```

**Comment by sateuwdie** [comment]
I use those commands

```
mkdir -p build
cd build

cmake ..     -DCMAKE_C_FLAGS:STRING="$SLKCFLAGS"     -DCMAKE_CXX_FLAGS:STRING="$SLKCFLAGS -std=c++11"     -DCMAKE_INSTALL_PREFIX=/usr     -DCMAKE_BUILD_TYPE=Release     $occ     -DBUILD_OSG_EXAMPLES=ON
make -j5
```

**Comment by HappySeaFox** [comment]
Hi! Thanks! Before the next release is out, you can fix it locally with the patch [19f340f](https://github.com/HappySeaFox/sail/commit/19f340fc72cf90a46f9efbe07ef3ce2c119ab22c)
